go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/buildpacks/imgutil v0.0.0-20240118145509-e94a1b7de8a9
 	github.com/buildpacks/lifecycle v0.18.5
 	github.com/docker/cli v25.0.3+incompatible
-	github.com/docker/docker v25.0.3+incompatible
```

Comment:
I think you need to update your fork, and rebase your branch to get the latest dependencies updates

Reply:
@jjbustamante codecov/patch demanding some test cases yet.. should we work on those or we can ignore that ?